scale-out-relay/Makefile

```diff
@@ -3,8 +3,10 @@
 # make licensed : will build a license limited version of the plugin
 #
 
-FILES =
+FILES = $(NAME)/server-node-to-relay
 SCRIPTS = postinst
 
 include ../makefiles/common-scala-plugin.mk
 
+target/$(NAME)/server-node-to-relay:
+	cp ./src/main/resources/server-node-to-relay target/$(NAME)/
```

Comment:
Missing final end of line.
Particularly important for old tools like make and the shell